### ISSUE TITLE
Unexpected rearrangement of comments in function containing imports

### ISSUE THREAD

**Issue by andersk** [timothycrosley/isort#1089]
isort (commit fbe1ab9290f08521d39a929b516a973b14a53627 from `develop`) unexpectedly reorders the comments in this code:

```diff
ERROR: /tmp/test.py Imports are incorrectly sorted and/or formatted.
--- /tmp/test.py:before	2020-01-07 17:13:44.571075
+++ /tmp/test.py:after	2020-01-07 17:13:45.654214
@@ -1,8 +1,9 @@
 def f():
     # comment 1
-    # comment 2
+
     # comment 3
     # comment 4
     from a import a
+    # comment 2
     from b import b
 
```

**Comment by timothycrosley** [comment]
Again thanks for reporting! I'll make sure to resolve this prior to release

~Timothy

**Comment by timothycrosley** [comment]
This is fixed in develop and will make it's way into 5.0.0 release :)
